plutus-core/plutus-ir/test/GeneratorSpec.hs

```diff
@@ -0,0 +1,23 @@
+-- editorconfig-checker-disable-file
+module GeneratorSpec where
+
+import Test.QuickCheck
+import Test.Tasty
+import Test.Tasty.Extras
+import Test.Tasty.QuickCheck
+
+import GeneratorSpec.Substitution
+import GeneratorSpec.Terms
+import GeneratorSpec.Types
+
+generators :: Int -> TestNested
+generators factor = return $ testGroup "generators"
+  [ testProperty "prop_genKindCorrect"  $ withMaxSuccess (factor*100000) (prop_genKindCorrect False)
+  , testProperty "prop_shrinkTypeSound" $ withMaxSuccess (factor*100000) prop_shrinkTypeSound
+  , testProperty "prop_genTypeCorrect"  $ withMaxSuccess (factor*10000) (prop_genTypeCorrect False)
+  , testProperty "prop_shrinkTermSound" $ withMaxSuccess (factor*20) prop_shrinkTermSound
+  , testProperty "prop_substType"       $ withMaxSuccess (factor*10000) prop_substType
+  , testProperty "prop_unify"           $ withMaxSuccess (factor*10000) prop_unify
```

Comment:
@MaximilianAlgehed there are 8 property tests in `GeneratorSpec.Terms`, should all of them appear here? Or why are they ignored?

Reply:
Most of them are there to help you debug when the main property fails. I haven't had time to look back at these now (sorry, a bit swamped) but that's most likely what's going on. Though if you have infinite compute for CI it's of course not a bad idea to just run them all...

Reply:
I've included most of them into the main test suite and tweaked them not to take much time. My reasoning is that there's always a chance someone will screw up a test without realizing it and leave the broken test there (dead code is generally bad) and that chance is 100%, because apparently I did screw up at least two tests while refactoring stuff.

Thanks for answering the question!